blob/azureblob/azureblob.go

```diff
@@ -589,9 +589,6 @@ func (b *bucket) ErrorAs(err error, i interface{}) bool {
 }
 
 func (b *bucket) ErrorCode(err error) gcerrors.ErrorCode {
-	if code := gcerrors.Code(err); code != gcerrors.Unknown {
```

Comment:
Note: I moved this to the concrete class (blob.go). If the driver produces an error that already has an error code, we should always just use it.